### ISSUE TITLE
md to docx: visual issues with line breaks in text + blockquotes

### ISSUE THREAD

**Issue by jwijay** [stencila/encoda#95]
The following visual issues occur when converting from `md` to `docx`.

Tested this conversion with the following `md` file: https://github.com/datacarpentry/R-ecology-lesson/blob/master/README.md

`npx ts-node --files src/cli convert examples/R-ecology-lesson/README.md ./examples/output/README.docx`

Output README.docx:
![Screenshot 2019-06-05 11 45 35](https://user-images.githubusercontent.com/10161095/58992625-31b56780-87a0-11e9-8931-19fcbf4d26e1.png)

**Issue 1:** Paragraphs on multiple lines in `md` (see below) get translated into spaced out paragraphs in `docx`. This becomes an issue when completing a roundtrip conversion back to `md`, since line breaks in `docx` get translated to ``, so words get unintentionally concatenated.
```
# Data carpentry: R for data analysis and visualization of Ecological Data

This is an introduction to R designed for participants with no programming
experience. These lessons can be taught in 3/4 of a day (approximately 6 hours). They start
with some basic information about syntax for the R programming language, the
RStudio interface, and move through to specific programming tasks, such as
importing CSV files, the structure of data frame objects in R, dealing with
categorical variables (i.e. factors), basic data manipulation (adding/removing
rows and columns), and finishing with calculating summary statistics and a brief
introduction to plotting. There is also a lesson on how to use databases from R that is intended to be taught after the SQL lesson, and ideally at the end of a Data Carpentry workshop.
```
**Result (`md` to `docx`):**
![Screenshot 2019-06-05 14 50 18](https://user-images.githubusercontent.com/10161095/58993061-49412000-87a1-11e9-9f00-cd723642543d.png)

**Result (completing roundtrip from `docx` to `md`:**
```
# Data carpentry: R for data analysis and visualization of Ecological Data

This is an introduction to R designed for participants with no programmingexperience. These lessons can be taught in 3/4 of a day (approximately 6 hours). They startwith some basic information about syntax for the R programming language, theRStudio interface, and move through to specific programming tasks, such asimporting CSV files, the structure of data frame objects in R, dealing withcategorical variables (i.e. factors), basic data manipulation (adding/removingrows and columns), and finishing with calculating summary statistics and a briefintroduction to plotting. There is also a lesson on how to use databases from R that is intended to be taught after the SQL lesson, and ideally at the end of a Data Carpentry workshop.
```

---

**Issue 2:** When a list item is present in a blockquote (see below), an extra bullet is added. When converting from `md`, extra spacing is added in blockquotes in the output `docx`.

```
## Prerequisites

> * Having R and RStudio installed (though see the first
> lesson, [Before we start](http://datacarpentry.org/R-ecology-lesson/00-before-we-start.html) for installation
> instructions)
```
**Result:**
![Screenshot 2019-06-05 14 50 11](https://user-images.githubusercontent.com/10161095/58993065-4fcf9780-87a1-11e9-8889-06ca97bbdbe8.png)

In both cases (line breaks in text and blockquotes), we might want to consider recognizing "paragraphs" from multiple lines, and in `docx` combine multiple lines into a single line or paragraph (unwrap). Then during conversion back to `md`, allow auto-wrap functionality (maybe enable by default).

**Comment by nokome** [comment]
Closed with https://github.com/stencila/encoda/pull/111 and another PR I can't find right now
